src/plugins/shared-variables/simulations/simulation-utilities.ts

```diff
@@ -7,3 +8,8 @@ export function isInputVariable(variable: VariableType) {
 export function isOutputVariable(variable: VariableType) {
   return variable.hasLabel("output");
 }
+
```

Comment:
@tealefristoe this was necessary because I didn't want to go and change the schema for the model to allow `decimalPlaces` to be a new field in the `Variable` type, but let me know if we should.  My assumption is that would be too deep a change and start messing with code shared with Diagram, etc.

Reply:
@bacalj It was good intuition to not want to go messing with quantity playground for something like this. But there's actually already a system that allows for this kind of customization in the variables. Labels of the form "type:value" can assign custom values for different variables, and it has the advantage of keeping all of a variable's specifications in one place rather than hiding some in a utility file.

I also made the default 2 decimal places rather than 0. 0 seems more clean and like a better default, but I think it'll be more clear to engineers in the future what's going on if they always see two decimal places rather than whole numbers, which might make it seem like the numbers aren't being rounded or truncated or anything.

Note that if you want to test this change, you'll have to start with a fresh document, because once the variables are created they won't be changed in a future.

Reply:
Great, makes sense.  Tested and it works.